scheduler-driver/src/test/java/gov/nasa/jpl/aerie/scheduler/TestUtility.java

```diff
@@ -146,4 +152,23 @@ public static Instant timeFromEpochDays(int days) {
     return timeFromEpochHours(days * 24);
   }
 
+  /** matches activities if they agree in everything except the (possibly auto-generated) names **/
+  public static void assertActivitiesListsEquals(List<SchedulingActivityDirective> expected, List<SchedulingActivityDirective> actual) {
```

Comment:
Can we change the name of this procedure by assertSetEquality ?

Reply:
Done 👍